examples/Extensions/AddSitelinks.php

```diff
@@ -0,0 +1,339 @@
+            ])
+        ]);
+
+        // Creates a site link feed item 4.
+        $sitelinkFeedItem4 =
+            self::createSitelinkFeedItem('Happy hours', 'http://www.example.com/happyhours');
+
+        // Creates an extension feed item from the sitelink feed item.
+        $extensionFeedItem4 = new ExtensionFeedItem([
+            'extension_type' => ExtensionType::SITELINK,
+            'sitelink_feed_item' => $sitelinkFeedItem4,
+            'targeted_campaign' => new StringValue(['value' => $campaignResourceName]),
+            'ad_schedules' => [
+                self::createAdScheduleInfo(
+                    DayOfWeek::MONDAY,
+                    18,
+                    MinuteOfHour::ZERO,
+                    21,
+                    MinuteOfHour::ZERO
+                ),
+                self::createAdScheduleInfo(
+                    DayOfWeek::TUESDAY,
+                    18,
+                    MinuteOfHour::ZERO,
+                    21,
+                    MinuteOfHour::ZERO
+                ),
+                self::createAdScheduleInfo(
+                    DayOfWeek::WEDNESDAY,
+                    18,
+                    MinuteOfHour::ZERO,
+                    21,
+                    MinuteOfHour::ZERO
+                ),
+                self::createAdScheduleInfo(
+                    DayOfWeek::THURSDAY,
+                    18,
+                    MinuteOfHour::ZERO,
+                    21,
+                    MinuteOfHour::ZERO
+                ),
+                self::createAdScheduleInfo(
+                    DayOfWeek::FRIDAY,
+                    18,
+                    MinuteOfHour::ZERO,
+                    21,
+                    MinuteOfHour::ZERO
+                )
+            ]
+        ]);
+
+        // Issues a mutate request to add the extension feed items.
+        $extensionFeedItemServiceClient = $googleAdsClient->getExtensionFeedItemServiceClient();
+        $response = $extensionFeedItemServiceClient->mutateExtensionFeedItems(
+            $customerId,
+            [
+                new ExtensionFeedItemOperation(['create' => $extensionFeedItem1]),
+                new ExtensionFeedItemOperation(['create' => $extensionFeedItem2]),
+                new ExtensionFeedItemOperation(['create' => $extensionFeedItem3]),
+                new ExtensionFeedItemOperation(['create' => $extensionFeedItem4])
+            ]
+        );
+
+        // Prints some information about the created extension feed items.
+        printf(
+            "Created %d extension feed items with the following resource names:%s",
+            $response->getResults()->count(),
+            PHP_EOL
+        );
+        $createdExtensionFeedItemsResourceNames = [];
+        foreach ($response->getResults() as $addedExtensionFeedItem) {
+            /** @var ExtensionFeedItem $addedExtensionFeedItem */
+            print "{$addedExtensionFeedItem->getResourceName()}" . PHP_EOL;
```

Comment:
`$addedExtensionFeedItem->getResourceName()` is called two times, you could use a variable here